Common/Qt/Options/EditableTable/EditableTableBaseOption.h

```diff
@@ -56,6 +56,7 @@ class EditableTableBaseOption{
 
     size_t size() const{ return m_current.size(); }
     const EditableTableRow& operator[](size_t index) const;
+    EditableTableRow& operator[](size_t index);

```

Comment:
Can I do this ? Else, I can let the config stay constant and have instead the amount of moves I did count up, until it reach the config